dataprofiler/profilers/float_column_profile.py

```diff
@@ -57,7 +78,39 @@ def __add__(self, other):
                                  other.__calculations)
 
         if "precision" in merged_profile.__calculations:
-            merged_profile.precision = min(self.precision, other.precision)
+
+            if self.precision['min'] is None:
+                merged_profile.precision = copy.deepcopy(other.precision)
+            elif other.precision['min'] is None:
+                merged_profile.precision = copy.deepcopy(self.precision)                
+            else:
+                merged_profile.precision['min'] = min(
+                    self.precision['min'], other.precision['min'])
+                merged_profile.precision['max'] = max(
+                    self.precision['max'], other.precision['max'])
+                merged_profile.precision['sum'] = \
+                    self.precision['sum'] + other.precision['sum']
+                merged_profile.precision['sample_size'] = \
+                    self.precision['sample_size'] + other.precision['sample_size']
+
+                merged_profile.precision['var'] = self._merge_variance(
+                    self.precision['sample_size'],
+                    self.precision['var'], self.precision['mean'],
+                    other.precision['sample_size'], other.precision['var'],
+                    other.precision['mean'])
+                merged_profile.precision['mean'] = \
+                    merged_profile.precision['sum'] \
+                    / merged_profile.precision['sample_size']
+            
+                merged_profile.precision['std'] = math.sqrt(
+                    merged_profile.precision['var'])
+
+                # Margin of error, 99.9% confidence level
+                merged_profile.precision['margin_of_error'] = \
+                    merged_profile.__z_value_precision * (
+                        merged_profile.precision['std']
+                        / math.sqrt(merged_profile.precision['sample_size'])
+                )
```

Comment:
in the `_update` sig figs are rounded but it doesn't look like it is done here as well.

Reply:
Removed the round in the other location - shouldn't be rounding pre-output.